### ISSUE TITLE
Add timezone command

### ISSUE THREAD

**Issue by MrPowerGamerBR** [LorittaBot/Loritta#2299]
Original Suggestion: https://canary.discord.com/channels/297732013006389252/359139508681310212/787223970175778816

This is a discussion issue, I will add a bounty after we figure out:
* How the command should work
* How the command should look

**Comment by iiAmFilippi** [comment]
I don't think it would be so useful for a command to show ONLY the hours of a certain place. This information could be added to other commands, such as "+weather".
